src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_load_store_emitters.cpp

```diff
@@ -0,0 +1,137 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "jit_load_store_emitters.hpp"
+#include "cpu/aarch64/cpu_isa_traits.hpp"
+
+using namespace Xbyak_aarch64;
+
+namespace ov {
+namespace intel_cpu {
+namespace aarch64 {
+
+using jit_generator = dnnl::impl::cpu::aarch64::jit_generator;
+using cpu_isa_t = dnnl::impl::cpu::aarch64::cpu_isa_t;
+
+jit_load_emitter::jit_load_emitter(dnnl::impl::cpu::aarch64::jit_generator *host, dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                   ov::element::Type src_prc, ov::element::Type dst_prc, int load_num, int byte_offset,
+                                   ov::element::Type exec_prc, emitter_in_out_map in_out_type)
+: jit_emitter(host, host_isa, exec_prc, in_out_type), name_("unknown"), load_num_(load_num), byte_offset_(byte_offset),
+              src_prc_(src_prc), dst_prc_(dst_prc) {
+    prepare_table();
+}
+
+void jit_load_emitter::emit_impl(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const {
+    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
+        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_idxs, out_idxs);
+    } else {
+        OPENVINO_THROW("Load emitter in ", name_, " is performed on unsupported isa.");
+    }
+}
+
+template <cpu_isa_t isa>
+void jit_load_emitter::emit_isa(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const {
+    bool matched_prc = src_prc_ == ov::element::f32 && dst_prc_ == ov::element::f32;
+    if (!matched_prc)
+        OPENVINO_THROW("Load emitter in ", name_, " only support both input and output precisions of being FP32.");
+    if (load_num_ > static_cast<int>((get_vec_length() / dst_prc_.size())))
+        OPENVINO_THROW("Load emitter in ", name_, " have unexpected number of elements to load.");
+
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+    XReg src = XReg(in_idxs[0]);
+    XReg prc = XReg(aux_gpr_idxs[0]);
+    TReg dst = TReg(out_idxs[0]);
+    SReg dst_s = SReg(out_idxs[0]);
+    DReg dst_d = DReg(out_idxs[0]);
+
+    h->add(prc, src, byte_offset_);
```

Comment:
`byte_offset_` can be any value. Use this approach, please, to load data with some offset in more secure way: https://github.com/openvinotoolkit/openvino/blob/cc128992e6741ad9b6bb9ba636a790252cc3414d/src/plugins/intel_cpu/src/nodes/kernels/aarch64/jit_uni_eltwise_generic.cpp#L288

If you find something more useful, then please share.

Reply:
Applied offset while loading/storing for in most cases. And applied Edward's approach for the case of h->ld1, because h->ld1 has more strict offset size check for element load. Thanks Alexandra and Edward!

Reply:
Yes, I verified all of these different store_num_ cases of load and store. They work.

Reply:
It's really great because we don't use aux gpr and don't need to make `push` `pop` for each `load`/`store` call 🔥 
Thanks!